adapters/gst/sources/rtsp.sh

```diff
@@ -35,6 +35,7 @@ SINK_PROPERTIES=(
     socket="${ZMQ_ENDPOINT}"
     sync="${SYNC_OUTPUT}"
     ts-offset="${SYNC_DELAY}"
+    eos-on-start="${EOS_ON_START}"
```

Comment:
There is no value by default. Have you checked the behavior if we don't provide EOS_ON_START?